### DISCUSSION TITLE
[Application Development] <Indexing Optimisim mainnet>

### DISCUSSION THREAD

**Discussion by d4v1d03** [ethereum-optimism/developers#496] [Application Development] [answered]
### Did you check the documentation?

- [X] Yes

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

hey i wanted to index the deposit and withdrawal transactions and was going through the docs of the standard bridge but couldn't find how to link source and destination data, or how what parameter to map data on source and destination chains.

**Comment by imtipi** [comment, marked as answer]
have you checked optimism SDK?might give you some idea
https://github.com/ethereum-optimism/optimism-tutorial/tree/main/sdk-view-tx (outdated! but you can see how it works)
